Tools/build/mypy.ini

```
@@ -1,7 +1,9 @@
 [mypy]
 files =
     Tools/build/compute-changes.py,
-    Tools/build/generate_sbom.py
+    Tools/build/generate_sbom.py,
+    Tools/build/update_file.py
```

Does it work with a final trailing comma? So the next addition is a single line diff:

```suggestion
    Tools/build/generate_sbom.py,
    Tools/build/update_file.py,
```

---

Nope :( 
This is not supported in 1.15, but will be supported in 1.16, I already fixed this on mypy's side.